test/scan/test_scan_spmd.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import os
+import re
+import unittest
+from pathlib import Path
+
+import torch
+import torch_xla
+from torch_xla.experimental.scan import scan
+from torch_xla.distributed.spmd import mark_sharding, set_global_mesh, Mesh
+import torch_xla.runtime as xr
+
+
+class ScanSpmdTest(unittest.TestCase):
+
+  def setUp(self):
+    # Set up a simple SPMD mesh for these tests.
+    num_devices = xr.global_runtime_device_count()
+    mesh_shape = (num_devices,)
+    self.spmd_mesh = Mesh(list(range(num_devices)), mesh_shape, ('model',))
```

Comment:
I think you can just do `xs.get_1d_mesh(axis_name="model")`

Reply:
done